doc/news/changes/minor/20170109Jean-PaulPelteret_1

```diff
@@ -0,0 +1,6 @@
+Fixed: A bug in the rotation matrix used in step-18 has now been corrected.
+This tutorial now uses the function
+Physics::Transformations::Angular::rotation_matrix_3d to compute the 3d rotation
+matrix.
+<br>
+(Jean-Paul Pelteret, 2017/01/09)
```

Comment:
Yes of course, that was an oversight on my part. @kuberry would you mind providing me with your name so that I can add it to the changelog?